js/company.js

```diff
@@ -5,15 +5,19 @@ class Company{
         this.pagesSeenOn = 0;
     };
 
-    increment(name){
-        this[name] += 1;
+    incrementCount(){
+        this.count += 1;
+    };
+
+    incrementPagesSeenOn(){
+        this.pagesSeenOn += 1;
     };
 
     get(name){
         return this[name];
     };
```

Comment:
Can you also fix the `get(name)` signature here since you changed `set(property, val)`? They should match. `get(name)` is not getting the class property name, it should read `get(property)`.